packages/file-settings/README.md

```diff
@@ -1,17 +1,47 @@
-# File Settings
+# `@frontity/file-settings`
 
-### _This is an internal package used by @frontity/core._
+[![Version](https://img.shields.io/npm/v/@frontity/file-settings.svg)](https://www.npmjs.com/package/@frontity/file-settings) [![npm](https://img.shields.io/npm/dw/@frontity/file-settings)](https://www.npmjs.com/package/@frontity/file-settings) [![License: Apache--2.0](https://img.shields.io/badge/license-Apache%202-lightgrey)](https://github.com/frontity/frontity/blob/master/LICENSE)
 
-## Usage
+> This is an internal package used by [@frontity/core](https://github.com/frontity/frontity/tree/dev/packages/core)
 
-This package is used to import the Frontity settings from a local file.
+A settings package for Frontity that reads a `frontity.settings.js` file.
+This package allows to decouple from where the settings are retrieved.
```

Comment:
It's not actually this package the one that lets you decouple, but Frontity itself. There will be more `xxx-settings` packages in the future, but so far that is not fully implemented so I will remove this part.